stacks-signer/src/signerdb.rs

```diff
@@ -543,6 +604,8 @@ mod tests {
         let db_path = tmp_db_path();
         let mut db = SignerDb::new(db_path).expect("Failed to create signer db");
         let (mut block_info, block_proposal) = create_block();
+        // We'll need the V1 data fields for this
+        block_info.ext = ExtraBlockInfo::V1(BlockInfoV1::default());
```

Comment:
`get_first_signer_block_in_tenure` is used in `v0` (as is the `signed_over` field)